src/data/examples/index.ts

```diff
@@ -4,13 +4,13 @@ export const DEFAULT_EXAMPLE_LANGUAGES = ['javascript', 'react'];
 
 export const examples: Example[] = [
   {
-    id: 'chat-online-status',
-    name: 'Online Status',
+    id: 'chat-presence',
```

Comment:
This is the reference to the folder, so needs to remain as `chat-online-status`.

Reply:
Ah, I'll just move the folder :D

Reply:
522a02e08f85af7f3e7d3b6abcc2de359134fd02